### ISSUE TITLE
Pass a root directory for cloud builds

### ISSUE THREAD

**Issue by tophtucker** [observablehq/feedback#649]
Currently, you can’t build a Framework project on Observable Cloud unless it’s at the root of the repository. It’d be nice if you could specify a root directory, e.g. here:

<img width="680" alt="image" src="https://github.com/user-attachments/assets/f2857756-f7c8-4334-b3e3-932131e3d04c">

**Comment by tophtucker** [comment]
Note that there’s already a concept of a [configurable root](https://observablehq.com/framework/config#root) in observablehq.config.js. But observablehq.config.js itself still has to be at the root.

**Comment by mbostock** [comment]
Yep, terminology-wise, this would be the “project root” whereas the **root** config option specifies the “source root” (which defaults to `src` within the project root).

**Comment by kylehodgson** [comment]
here to upvote... is this reply what's needed, or is there something more formal?

**Comment by mbostock** [comment]
@kylehodgson Click the little smiley face 🙂 on the top post and then click the thumbs up 👍 to upvote.
